lms/djangoapps/teams/static/teams/js/spec/teams_spec.js

```diff
@@ -9,13 +9,38 @@ define([
                     return {
                         name: "team " + i,
                         id: "id " + i,
-                        language: "English",
-                        country: "Sealand",
+                        language: languages_ary[i%4][0],
+                        country: countries_ary[i%4][0],
                         is_active: true,
                         membership: []
                     };
                 });
-            };
+            },
+            countries_ary = [
```

Comment:
Nit: we try to avoid abbreviations in variable names. I would expect just "countries" or "languages".